charts/datadog/templates/_helpers.tpl

```diff
@@ -503,3 +503,14 @@ true
 false
 {{- end -}}
 {{- end -}}
+
+{{/*
+Return true if the Cluster check runners need a confd configmap
+*/}}
+{{- define "need-clusterchecks-confd" -}}
+{{- if (or .Values.datadog.kubeStateMetricsCore.enabled) -}}
+true
+{{- else -}}
+false
+{{- end -}}
+{{- end -}}
```

Comment:
Not sure why we'd need this 🤔  The cluster check runners must run the check as a cluster check (dispatched by the DCA) not with static config. Am I missing something?